wsmaster/che-core-api-workspace-activity/src/test/java/org/eclipse/che/api/workspace/activity/WorkspaceActivityManagerTest.java

```diff
@@ -71,31 +71,11 @@ private void setUp() throws Exception {
   public void shouldAddNewActiveWorkspace() throws Exception {
     final String wsId = "testWsId";
     final long activityTime = 1000L;
-    final Map<String, Long> activeWorkspaces = getActiveWorkspaces(activityManager);
-    boolean wsAlreadyAdded = activeWorkspaces.containsKey(wsId);
 
     activityManager.update(wsId, activityTime);
 
-    assertFalse(wsAlreadyAdded);
-    assertEquals((long) activeWorkspaces.get(wsId), activityTime + EXPIRE_PERIOD_MS);
-    assertFalse(activeWorkspaces.isEmpty());
-  }
-
-  @Test
-  public void shouldUpdateTheWorkspaceExpirationIfItWasPreviouslyActive() throws Exception {
```

Comment:
Can you elaborate on why we don't need this test anymore?

Reply:
Yes. Since manager is not holding any data anymore, there is no more difference between creating or updating existing data. The test will look pretty the same, so no big reason to write it.

Reply:
ok, thank you for the explanation